packages/api/src/db/schemas/batchTopic.js

```diff
@@ -26,6 +26,16 @@ const batchTopicSchema = new Schema({
     type: Schema.Types.ObjectId,
     required: true,
   },
+
+  masterId: { // link to 'master topic' collection
```

Comment:
Didn't we discuss that we would de-link the batchX (X === exercises/topics) from masterX, to allow instructors to change batchX after batch creation?

Reply:
That's why we have a copy of all fields in `BatchTopic` collection, but we need to have the reference for master collection.

Reply:
We need to know what is included in a particular batch, without reference, it's hard to determine that.